uktrade/great-cms
# Make focus outline on filter radios/checkboxes fully visible

This PR fixes a styling bug with the filters displayed at `/markets` and `/export-academy/events` where the yellow focus around the checkboxes and radio buttons was cut off.

**To test**
- Go to `/markets` or `/export-academy/events`
- Click on the radios/checkboxes and check that the yellow outline isn't cut off

**Screenshots**

**Before**
![Screenshot 2023-05-02 at 17 09 41](https://user-images.githubusercontent.com/22460823/235723189-da839588-1435-45cd-a4d1-1c03d322a574.png)

**After**
![Screenshot 2023-05-02 at 17 10 39](https://user-images.githubusercontent.com/22460823/235723414-a9c787b1-8211-4e91-bde3-c1edab0f77da.png)

### Workflow

- [ ] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-538

### Reviewing help


### Housekeeping


### Merging
